doc-style/action.yml

```diff
@@ -4,7 +4,7 @@ description: "Check documentation style using Vale"
 inputs:
   vale-config:
     description: "Path to the Vale configuration file"
-    default: "doc/.vale.ini"
+    default: 'doc/.vale.ini'
```

Comment:
I prefer to use of the double quotations--mainly because that is what Alex told me to use when I did the overall edit Of PyAEDT before its public debut.

Reply:
Sure, it should be okay to apply those here. Let me remove the changes in this line.

The syntax for GitHub actions can be tricky. Some parameters need to use single quotes. Otherwise, they are badly interpreted. As said, I bet this one can make use of double quotes.